patterns/footer-large.php

```diff
@@ -14,20 +14,14 @@
 		<!-- wp:column {"width":"60%","style":{"spacing":{"padding":{"right":"50px"}}}} -->
 		<div class="wp-block-column" style="padding-right:50px;flex-basis:60%">
 			<!-- wp:group {"style":{"spacing":{"blockGap":"8px"}},"layout":{"type":"flex","orientation":"vertical"}} -->
-			<div class="wp-block-group">
-				<!-- wp:site-logo /-->
+			<div class="wp-block-group"><!-- wp:site-logo /-->
+				<!-- wp:heading {"level":5,"style":{"typography":{"textTransform":"none"},"spacing":{"margin":{"top":"40px"}}}} -->
+				<h5 class="wp-block-heading" style="margin-top:40px;text-transform:none">Join the community</h5>
+				<!-- /wp:heading -->
 
-				<!-- wp:group {"style":{"spacing":{"margin":{"top":"40px","bottom":"40px"},"blockGap":"12px"}},"layout":{"type":"constrained"}} -->
-				<div class="wp-block-group" style="margin-top:40px;margin-bottom:40px">
-					<!-- wp:heading {"level":5,"style":{"typography":{"textTransform":"none"}}} -->
-					<h5 class="wp-block-heading" style="text-transform:none"><?php esc_html_e( 'Join the community', 'woo-gutenberg-products-block' ); ?></h5>
-					<!-- /wp:heading -->
-
-					<!-- wp:paragraph -->
-					<p><?php esc_html_e( 'Learn about new products and discounts', 'woo-gutenberg-products-block' ); ?></p>
-					<!-- /wp:paragraph -->
-				</div>
-				<!-- /wp:group -->
+				<!-- wp:paragraph {"style":{"spacing":{"margin":{"bottom":"40px"}}}} -->
+				<p style="margin-bottom:40px">Learn about new products and discounts</p>
```

Comment:
I think that you forgot to use: `esc_html_e`.

Reply:
Good catch! Copy pasting... 😅